### ISSUE TITLE
The list command with  '--plain --limit X' does not respect numbers bigger then 100

### ISSUE THREAD

**Issue by bazulay** [ankitpokhrel/jira-cli#352]
**Describe the bug**

when running the following command I only get the first 100 results
```
jira issue list -q "fixVersion = 2.0.0" --plain --limit 1000
```

1. JiraCLI Version:
```
(Version="0.3.0", GitCommit="ea53355d5b72994a5b17a63e6e800ce9d8a23f4e", CommitDate="2022-01-09T19:50:19+00:00", GoVersion="go1.17.5", Compiler="gc", Platform="linux/amd64")
```

3. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    cloud
    ```
4. What operating system are you using? Also mention version.
    ```
    Linux Fedora 35 
    ```
5. What terminal are you using? Also mention version.
    ```
   GNOME Terminal Version 3.42.2 for GNOME 41
    ```

**To Reproduce**

1. perform the above (adjusted query) on any list bigger then 100

**Expected behavior**

when in --plain to respect given --limit

**Comment by ankitpokhrel** [comment]
Hi @bazulay, This is how Jira API behaves and we won't get expected number of results if the response body is huge. So `--limit` option is unreliable at the moment and needs to be changed to accept a range like a pagination.

[Doc](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-search/#api-rest-api-3-search-get) for ref:

![Screen Shot 2022-04-27 at 8 21 41 AM](https://user-images.githubusercontent.com/2364546/165454240-11e1c673-a885-4f06-9d21-29b63ecdc746.png)

Somewhat related: #277

**Comment by bazulay** [comment]
Thanks @ankitpokhrel 

What about pagination ? 
If we have pagination I can loop through all the pages .

**Comment by ankitpokhrel** [comment]
Fixed in #359

You can now use the `--paginate` flag. Format `<from>:<limit>`, where `<from>` is optional and `<limit>` must be between 1 and 100 (inclusive).

```sh
# Get 10 results from the start
$ jira issue list --paginate 10

# Get 50 results starting from position 10
$ jira issue list --paginate 10:50
```

**Comment by bazulay** [comment]
Thanks @ankitpokhrel  for the fast response.
When is the next release ? or where can I find instructions on howto build from source ?
